src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -163,7 +163,7 @@ export class FollowArtistButton extends React.Component<Props, State> {
 
     // Custom button renderer
     const content = render ? (
-      <span onClick={this.handleFollow}> {render(artist)}</span>
+      <Clickable onClick={this.handleFollow}> {render(artist)}</Clickable>
     ) : (
```

Comment:
So the annoying thing here is that the spec failures are due to nested buttons. I've run into this elsewhere — there's a lot of components around the site with weirdly nested onClick handlers.

One way of getting around this is rather than using the `Clickable` button, keep the span but add `tabIndex={0}` to allow it to be focusable. Then adding the `onKeyPress` action to support it. An example is here:
https://github.com/artsy/force/blob/f6a3519c4f1df7879dd44ce9cbd9faa2e1189909/src/v2/Components/NavBar/Menus/UserMenu.tsx#L104-L109\

Reply:
It should be noted that I do think this is more of a treating the symptom kind of thing. Ideally there should be any nested  click handlers like that in the first place.